subprojects/gradle/instrumentation-tests-default-config/src/main/kotlin/com/avito/instrumentation/InstrumentationDefaultConfigPlugin.kt

```diff
@@ -57,6 +57,11 @@ class InstrumentationDefaultConfigPlugin : Plugin<Project> {
                     it.fromSource.includeByAnnotations(TestsFilter.uiNoE2E.annotatedWith)
                     it.fromRunHistory.excludePreviousStatuses(setOf(RunStatus.Success, RunStatus.Manual))
                 }
+                filters.register("uiNoE2eNoFlaky") {
+                    it.fromSource.includeByAnnotations(TestsFilter.uiNoE2E.annotatedWith)
+                    it.fromSource.excludeByAnnotations(TestsFilter.flaky.annotatedWith)
+                    it.fromRunHistory.excludePreviousStatuses(setOf(RunStatus.Success, RunStatus.Manual))
+                }
```

Comment:
What about reusing filters?
We've talked about something like `filter.initWith(anotherFilter)` to copy an existing filter.
It'd better show the intention to add only one annotation without duplication.

Reply:
not a blocker

Reply:
I've tried to make something like `initWith(anotherFilter)` but current Filter implementation always rewrite its internal Sets like the following:
```kotlin
fun include(values: Set<T>) {
        included = values
}
```

So if I even init some filter with another I won't be able to include or exclude something because I'll overwrite values copied from another filter.

@sboishtyan fyi